### PR TITLE
Bulletin tweaks

### LINKED ISSUE #2777: Bulletin tweaks
**Is your feature request related to a problem? Please describe.**

Following the UX review on the TV/Radio bulletin component, we’d need to capture the following changes:

- Radio and TV Bulletin 240px -399px
   - Margin between image and headline to be 8px
headline-padding

   - Adjust the size of SVG to be 16px height and review alignment of the PlayCTA icon and text

  -  Change Headline typography from `Pica` to `GreatPrimer` above 600px

**Describe the solution you'd like**

 - In **headingStyles**  replace
   `padding: ${GEL_SPACING_DBL} ${GEL_SPACING} ${GEL_SPACING} ${GEL_SPACING};` 
   with 
   `padding: ${GEL_SPACING}`

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by ghost**
Resolves #2777

**Overall change:**
Apply some CSS tweaks suggested by UX to the Bulletin.

**Code changes:**

- Apply `GEL_SPACING` on heading
- Align the `PlayCTA` icon vertically
- Increase the size of the icon to 16px
- Apply `padding-right` to the Radio heading above 600px
- Change Headline typography from `Pica` to `GreatPrimer` above 600px

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-bulletin/src/index.jsx [hunk 1]
```diff
@@ -121,8 +121,10 @@ const IconWrapper = styled.span`
   > svg {
     color: ${C_WHITE};
     fill: currentColor;
+    width: 1.0625rem;
+    height: 1rem;
```
Comment (DenisHdz) [inline packages/components/psammead-bulletin/src/index.jsx]:
```suggestion
    height: {GEL_SPACING_DBL};
```

**Review by DenisHdz — CHANGES_REQUESTED** [review]
Looking at this PR I realized we should also apply padding-right to the **Radio** heading above 600px as we did with the summary https://github.com/bbc/psammead/pull/2760, otherwise, we could have cases like this, where the text is attached to the border:

radio-headline

**Comment by ghost** [comment]
@DenisHdz I have added the padding to `RadioHeading` and also handled `rtl`.

**Review by DenisHdz — APPROVED** [review]
Thanks for the changes. Please make sure to review it with UX before merge it

File: packages/components/psammead-bulletin/src/index.jsx [hunk 2]
```diff
@@ -183,8 +184,10 @@ const TVHeading = styled.h3`
 const RadioHeading = styled.h3`
   ${headingStyles}
   @media(min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    padding: ${GEL_SPACING} 0;
+    padding-top: ${GEL_SPACING};
+    padding-bottom: ${GEL_SPACING};
   }
+  ${({ dir }) => (dir === 'ltr' ? `padding-left: 0;` : `padding-right: 0;`)}
```
Comment (AlistairGempf) [inline packages/components/psammead-bulletin/src/index.jsx]: image
The heading of the radio bulletin should presumably have some left padding?
Does this line need to be moved into the media query above?

Comment (ghost) [reply packages/components/psammead-bulletin/src/index.jsx]: @AlistairGempf Nice spot. Moved it to the media query above

**Review by AlistairGempf — APPROVED** [review]
Looks good

**Comment by DenisHdz** [comment]
I have been working on aligning vertically the icon and text in the `PlayCTA`, but after trying several CSS combinations, I didn't manage to center the text.
I initially thought that the issue was with the `line-height` applied by each service, but I think it might be caused by the `font-family` used by the service because as you can see in the video, it looks fine for News, but when selecting Tamil or Tigrinya, the text moves up or down when checking and unchecking the font-family.

cta-text-trim

I have been looking at how we define the font-faces in the `fonts.js` and everything looks fine.
On top of that, each browser renders these fonts in its own way with a different vertical top and bottom offset.

**Comment by DenisHdz** [comment]
> I have been working on aligning vertically the icon and text in the `PlayCTA`, but after trying several CSS combinations, I didn't manage to center the text.
> I initially thought that the issue was with the `line-height` applied by each service, but I think it might be caused by the `font-family` used by the service because as you can see in the video, it looks fine for News, but when selecting Tamil or Tigrinya, the text moves up or down when checking and unchecking the font-family.
> 
> cta-text-trim
> 
> I have been looking at how we define the font-faces in the `fonts.js` and everything looks fine.
> On top of that, each browser renders these fonts in its own way with a different vertical top and bottom offset.

After reviewing it with UX, we have decided to create another [issue](https://github.com/bbc/psammead/issues/2881) to deal with this alignment bug, so we can merge the rest of the tweaks on this PR.

**Review by AlistairGempf — APPROVED** [review]
Rereviewing - looks good.

**Review by tochwill — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
Looks good to me..

**Pull request merged**